vowpalwabbit/parse_example_json.h

```diff
@@ -1628,14 +1637,16 @@ struct VWReaderHandler : public rapidjson::BaseReaderHandler<rapidjson::UTF8<>,
 {
   Context<audit> ctx;
 
-  void init(vw* all, v_array<example*>* examples, rapidjson::InsituStringStream* stream, const char* stream_end,
+  void init(const label_parser& lbl_parser, hash_func_t hash_func, uint64_t hash_seed, uint64_t parse_mask,
```

Comment:
this is getting big, wondering if we should define a specific passthrough struct for all these arguments

Reply:
Agreed, if you are okay with it it will come along in another PR